projects/igniteui-angular/src/lib/combo/combo.component.html

```diff
@@ -23,33 +23,35 @@
             <ng-container *ngTemplateOutlet="clearIconTemplate"></ng-container>
         </ng-container>
         <igx-icon *ngIf="!clearIconTemplate">
-            clear
+            {{ inputGroup.theme === 'material' ? 'cancel' : 'clear'  }}
         </igx-icon>
     </igx-suffix>
     <igx-suffix class="igx-combo__toggle-button">
         <ng-container *ngIf="toggleIconTemplate">
             <ng-container *ngTemplateOutlet="toggleIconTemplate; context: {$implicit: this.collapsed}"></ng-container>
         </ng-container>
         <igx-icon *ngIf="!toggleIconTemplate">
-            {{ dropdown.collapsed ? 'arrow_drop_down' : 'arrow_drop_up'}}
+            {{ inputGroup.theme === 'material' ? dropdown.collapsed ? 'expand_more' : 'expand_less' : dropdown.collapsed ? 'arrow_drop_down' : 'arrow_drop_up' }}
         </igx-icon>
     </igx-suffix>
 </igx-input-group>
 <igx-combo-drop-down #igxComboDropDown class="igx-combo__drop-down" [displayDensity]="displayDensity"
     [labelledBy]="this.ariaLabelledBy || this.label?.id || this.placeholder || ''"
     [width]="itemsWidth || '100%'" (opening)="handleOpening($event)" (closing)="handleClosing($event)"
     (opened)="handleOpened()" (closed)="handleClosed()">
-    <igx-input-group *ngIf="displaySearchInput" [displayDensity]="displayDensity" theme="material" class="igx-combo__search">
-        <input class="igx-combo-input" igxInput #searchInput name="searchInput" autocomplete="off" type="text"
-            [(ngModel)]="searchValue" (ngModelChange)="handleInputChange($event)" (keyup)="handleKeyUp($event)"
-            (keydown)="handleKeyDown($event)" (focus)="dropdown.onBlur($event)" [attr.placeholder]="searchPlaceholder"
-            aria-autocomplete="list" role="searchbox" aria-label="search"/>
-        <igx-suffix *ngIf="showSearchCaseIcon">
-            <igx-icon family="imx-icons" name="case-sensitive" [active]="filteringOptions.caseSensitive"
-                (click)="toggleCaseSensitive()">
-            </igx-icon>
-        </igx-suffix>
-    </igx-input-group>
+    <div class="igx-combo__search">
+        <igx-input-group *ngIf="displaySearchInput" [displayDensity]="displayDensity" theme="material" >
+            <input igxInput #searchInput name="searchInput" autocomplete="off" type="text"
+                   [(ngModel)]="searchValue" (ngModelChange)="handleInputChange($event)" (keyup)="handleKeyUp($event)"
+                   (keydown)="handleKeyDown($event)" (focus)="dropdown.onBlur($event)" [attr.placeholder]="searchPlaceholder"
+                   aria-autocomplete="list" role="searchbox" aria-label="search"/>
+            <igx-suffix *ngIf="showSearchCaseIcon" (click)="toggleCaseSensitive()">
+                <span class="{{filteringOptions.caseSensitive? 'igx-combo__case-icon--active' : 'igx-combo__case-icon'}}">
```

Comment:
If we want to conditionally apply a class, we may want to use the `ngClass` directive or we could just move this whole thing in the component in a public, hidden/internal getter.